src/styles.css

```diff
@@ -78,3 +78,45 @@
 .location-submit-btn {
   width: 21.75rem;
 }
+
+.location-card {
+  max-width: 850px;
+}
+
+.location-radio-group {
+  margin: 0 auto;
+  background-color: var(--omrs-color-bg-high-contrast);
+  border: 2px solid rgba(249, 249, 249, 0.94);
+  list-style: none;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+}
+
+.location-radio-group input[type="radio"] {
+  opacity: 0;
+  position: fixed;
+  width: 0;
+}
+
+.location-radio-group label {
+  display: flex;
+  background-color: var(--omrs-color-bg-high-contrast);
+  color: var(--omrs-color-primary-blue);
+  padding: 5px;
+  font-family: sans-serif, Arial;
+  font-size: 12.8px;
```

Comment:
We use `rem` instead of `px`. 1rem = 16px so this would be 12.8/16 rem.

Reply:
@fatmali thanks.  I looked in the styleguide when I put this together and didn't see anything at all related to font sizes.  There don't seem to be any defined sizes to use throughout the application, and there is no mention of rem vs px vs other options.  Would you be able to update the style guide with the guidelines that we should be following?  
Also, the reason I chose 12.8px here is because in the Figma from Greg, when I click on these, that is the size that is listed in the style to the right.  It would be great if we are putting the effort into the Figmas if we could just use those styles as described.  Thoughts on that?

Reply:
Actually, now that I look again I see them.  Must have missed these before...

Reply:
Removed these font-family and font-size settings, in favor of defaults